libbeat/logp/config.go

```diff
@@ -59,6 +60,14 @@ type FileConfig struct {
 	RedirectStderr  bool            `config:"redirect_stderr" yaml:"redirect_stderr"`
 }
 
+// MetricsConfig contains configuration used by the monitor to output metrics into the logstream.
+//
+// Currently these options are not used through this object in beats (as monitoring is setup elsewhere).
+type MetricsConfig struct {
+	Enabled bool          `config:"enabled" yaml:"enabled"`
```

Comment:
@urso, I noticed that the beats load through https://github.com/elastic/beats/blob/60e9e348b9aef83e3addf997a5efede05821fa23/libbeat/cmd/instance/beat.go#L453

but the logging config does define these variables, do you think it makes sense to define it as part of the logging config so it can be retrieved by the agent?

Reply:
+1